.github/workflows/stalebot.yaml

```
@@ -0,0 +1,34 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-message: 'This issue has been automatically marked as stale due to lack of activity. You can remove the stale label or comment. Otherwise, this issue will be closed in 7 days. Thank you!'
+          close-issue-message: 'We are closing this issue due to lack of activity. Feel free to reopen it if you can provide more information. Thank you!'
+       
+          # Don't process PRs 
+          days-before-stale: -1
+
+          # Process only issues
+          days-before-issue-stale: 14
+          days-before-issue-close: 7
+          
+          # Add this label after 'exempt-issue-labels' days to mark it as stale
+          stale-issue-label: 'no-issue-activity'
```

Nit: You can re-use this for pull requests in the future and there is no need to have different labels for pull requests and issues.
```suggestion
          stale-issue-label: 'no-activity'
```

---

Good point, it is changed in the PR.